### PR TITLE
Updating TAG liaisons

### PR DESCRIPTION
**Pull request by amye**

### CODE REVIEW

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Comment by dims** [comment]
LGTM (non-binding!)

File: tags/README.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ a pull request with document referencing the roles and charter, updating the lis
...
-| [TAG Security](https://github.com/cncf/tag-security) | Emily Fox, Justin Cormack |
-| [TAG Storage](https://github.com/cncf/tag-storage) | Erin Boyd, Richard Hartmann |
-| [TAG App Delivery](https://github.com/cncf/tag-app-delivery) | Matt Farina, Lei Zhang, Katie Gamanji |
-| [TAG Network](https://github.com/cncf/tag-network) | Dave Zolotusky, Davanum Srinivas |
-| [TAG Runtime](https://github.com/cncf/tag-runtime) | Ricardo Rocha, Richard Hartmann, Davanum Srinivas |
-| [TAG Contributor Strategy](https://github.com/cncf/tag-contributor-strategy) | Matt Farina, Katie Gamanji, Cathy Zhang |
-| [TAG Observability](https://github.com/cncf/tag-observability) | Lei Zhang, Cathy Zhang |
-| [TAG Environmental Sustainability](https://github.com/cncf/tag-env-sustainability/) | Erin Boyd |
+| [TAG Security](https://github.com/cncf/tag-security) | Duffie Cooley, Dave Zolotusky|
+| [TAG Storage](https://github.com/cncf/tag-storage) | Emily Fox, Nikhita Raghunath  |
```
Comment (TheFoxAtWork) [inline tags/README.md:19]:
```suggestion
| [TAG Storage](https://github.com/cncf/tag-storage) | Matt Farina, Nikhita Raghunath  |
```

File: tags/README.md [hunk 2]
```diff
@@ -13,16 +13,16 @@ a pull request with document referencing the roles and charter, updating the lis
...
+| [TAG App Delivery](https://github.com/cncf/tag-app-delivery) | Katie Gamanji, Justin Cormack |
+| [TAG Network](https://github.com/cncf/tag-network) | Duffie Cooley, Ricardo Rocha |
+| [TAG Runtime](https://github.com/cncf/tag-runtime) | Nikhita Raghunath, Richard Hartmann, Cathy Zhang |
+| [TAG Contributor Strategy](https://github.com/cncf/tag-contributor-strategy) | Dave Zolotusky, Katie Gamanji |
```
Comment (TheFoxAtWork) [inline tags/README.md:23]:
```suggestion
| [TAG Contributor Strategy](https://github.com/cncf/tag-contributor-strategy) | Dave Zolotusky, Emily Fox |
```

**Review by TheFoxAtWork — COMMENTED** [review]
adjustment

**Pull request merged**